.circleci/workflows/openlineage-spark.yml

```diff
@@ -46,6 +46,7 @@ workflows:
           requires:
             - approval-integration-spark
       - configurable-integration-test-spark:
+          context: integration-tests
```

Comment:
otherwise we end up with: https://app.circleci.com/pipelines/github/OpenLineage/OpenLineage/11214/workflows/63ffa876-b03a-4da0-9b83-d7a2d6999545